src/devices/Ad7193/Ad7193.cs

```diff
@@ -0,0 +1,838 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Device.Spi;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using global::Iot.Units;
+
+namespace Iot.Device.Ad7193
+{
+    /// <summary>
+    /// Represents the Analog Devices AD7193, the 4-channel, 4.8 kHz, ultralow noise, 24-bit sigma-delta ADC with PGA
+    /// </summary>
+    public class Ad7193 : IDisposable
+    {
+        #region Metadata for IDevice
```

Comment:
We usually don't include the below members for metadata and instead just add them to the Readme file. There is no real need to have them as public API when they are really just info about the device, so they should really instead just be info in a Readme file.

Reply:
Is there any chance that you will include metadata in the classes themselves? I think they are very useful from the user's perspective, just like in the case of any Windows driver.
This would make it possible to compare different devices, ADCs programmatically, or list all supported devices in the framework, programmatically.
I also use KiCad EDA (a PCB designer program), and they store these pieces of information in their files as well, which is very useful.

Reply:
It would be good to have an actual scenario or need for the APIs first before we expose all these methods that we will then have to support in future versions. That said, if you think it is valuable to keep them, I would much rather have just one method called somehting like GetDeviceMetadata() which would in turn return an object that contained all of these properties. That way you don't polute your surface area, and you keep it simple so that it is a class that is easy to  use and interact with without the need to look into samples and such.